box.json.dist

```diff
@@ -14,7 +14,8 @@
         {
             "name": [
                 "*.php",
-                "*.php8"
+                "*.php8",
+                "completion.*"
```

Comment:
where are those files coming from? I don't see them in PR

Reply:
They are part of symfony/console